components/Auction/index.js

```diff
@@ -59,9 +41,26 @@ const HandleAuctions = () => {
       }
     } else {
       setUserMoney(0);
+      return null;
     }
   };
 
+  useEffect(() => {
+    (async () => {
```

Comment:
Please create named functions!
Reading and understanding stack traces would be far easier and better 😄 
```
async function funcname = () => {}
funcname()
```

Reply:
It's an IIFE ,please recheck the implementation,thanks